src/com/raghav/javabasics/DisplayArrayNumbersAreDivisible.java

```diff
@@ -0,0 +1,29 @@
+package com.raghav.javabasics;
+import java.util.Arrays;
+
+//Given an array of numbers, write a program to identify all numbers divisible by 2,
+// divisible by 3, divisible by 5, divisible by 7, divisible by 5 and divisible by 7.
+
+public class DisplayArrayNumbersAreDivisible {
+    public static void main(String[] args) {
+
+        DisplayArrayNumbersAreDivisible result = new DisplayArrayNumbersAreDivisible();
+        result.divisibilityChecker(new int[]{1,2,3,4,5,6,7,8,9,35});
+        result.divisibilityChecker(new int[]{});
+    }
+
+    public void divisibilityChecker(int[] myArray) {
+        int[] divNumArr = {2, 3, 5, 7, 35};
```

Comment:
Why 35? Where did this number come from?

Reply:
number divisible by 5 and 7

Reply:
Try to incorporate 5 and 7 in the code for 5 or 7.

if divisible by 5:
    if divisible by 7:
        both
    else:
        only 5
elif by 7:
     only 7